Update 3-printing.md

Henkjan1110 opened the pull request:
ChatGPT suggestions! Laat maar weten of er iets aangepast moet worden!

Henkjan1110 committed:
Update 3-printing.md

ChatGPT suggestions! Laat maar weten of er iets aangepast moet worden!

nickwkoning commented on data/week1/3-printing.md:
```diff
@@ -24,19 +26,19 @@ Hello World!
 </sample-output>
 
-The input that we give between brackets is referred to as an `argument` of the function. In the first case, we gave as argument to the function `System.out.println` the text ``"Hello World!"``, while in the second case our argument was ``"Goodbye!"``. Note that we can use any argument to this function, as long as it is composed of text, as indicated by the quotation marks `""` around the text tha ...
+The text within the brackets of a function (in this case `System.out.println`) is referred to as an `argument` and can be any text enclosed in quotation marks `""`. Our first example used `"Hello World!"` as an argument, while the second used `"Goodbye!"`.
```
Kan de suggestie wekken dat elk argument tussen aanhalingstekens moet. Misschien zo:

The text within the brackets of a function (in this case `System.out.println`) is referred to as an `argument` and can be any text. Our first example used `"Hello World!"` as an argument, while the second used `"Goodbye!"`. But we can use any text, as long as long as it is enclosed in quotation marks `""`.

nickwkoning reviewed (changes requested):
Eén opmerking. Voor de rest helemaal mooi!

Henkjan1110 committed:
Update 3-printing.md

incorporated feedback!

nickwkoning reviewed (changes requested):
Volgens mij gaat er nog iets mis en staat de tekst er nu 2x in en is de header van de exercise nu weggevallen? Ook enkele stukjes moeten binnen `` quotaties!

Henkjan1110 committed:
Update 3-printing.md

Ik zie inderdaad dat de aanpassingen niet goed zijn gegaan - vreemd.... Is het zo beter?

nickwkoning committed:
Update 3-printing.md
